pkg/resiliency/resiliency.go

```diff
@@ -292,7 +292,7 @@ func FromConfigurations(log logger.Logger, c ...*resiliencyV1alpha.Resiliency) *
 		log.Infof("Loading Resiliency configuration: %s", config.Name)
 		log.Debugf("Resiliency configuration (%s): %+v", config.Name, config)
 		if err := r.DecodeConfiguration(config); err != nil {
-			log.Errorf("Could not read resiliency policy %s: %w", &config.ObjectMeta.Name, err)
+			log.Errorf("Could not read resiliency policy %s: %w", config.ObjectMeta.Name, err)
```

Comment:
Seems not related to the topic.

Reply:
Happy to revert it, but this is currently printing the pointer address to a string, rather than the actual string value .

Reply:
Good catch, you can post another PR to fix.